From 8e8e3b069e9407e1fbe563f12f4269fe5d800621 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sun, 29 Oct 2023 00:37:33 -0300
Subject: [PATCH] Don't resize on Redmi

It breaks access to system partition unless we mount/unmount manually, leading to bootloop after a reboot
---
 rw-system.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rw-system.sh b/rw-system.sh
index 83a09e4..55f585f 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -341,7 +341,7 @@ if [ "$(getprop ro.product.vendor.manufacturer)" = motorola ] && getprop ro.vend
     setprop persist.sys.overlay.devinputjack true
 fi
 
-if ! getprop ro.vendor.build.fingerprint |grep samsung/;then
+if ! getprop ro.vendor.build.fingerprint | grep -iq -e samsung/ -e Redmi/; then
     if mount -o remount,rw /system; then
         resize2fs "$(grep ' /system ' /proc/mounts | cut -d ' ' -f 1)" || true
     else
-- 
2.42.0

